dtests/src/test/scala/io/snappydata/hydra/ct/CTTestUtil.scala

```diff
@@ -223,7 +224,8 @@ object CTTestUtil {
  */
 
   def addDataUsingPutInto(snc: SnappyContext): Unit = {
-    CTQueries.orders_details_df(snc).dropDuplicates("SINGLE_ORDER_DID").write.putInto("orders_details")
+    CTQueries.orders_details_df(snc).dropDuplicates("SINGLE_ORDER_DID")
```

Comment:
Please remove formatting changes in this method. I have modified the code for fixing a test issue with concurrent putInto.